Runtime/ui/painting/txt/mesh_generator.cs

```diff
@@ -99,24 +97,22 @@ class MeshInfo : PoolObject {
         }
     }
 
-    
+
     class TextBlobMesh : PoolObject {
-        
         static readonly Dictionary<MeshKey, MeshInfo> _meshes = new Dictionary<MeshKey, MeshInfo>();
         static long _frameCount = 0;
-        
+
         public TextBlob textBlob;
         public float scale;
         public uiMatrix3 matrix;
-        
+
         uiMeshMesh _mesh;
         bool _resolved;
 
         public TextBlobMesh() {
         }
 
         public override void clear() {
-            this.textBlob = null;
```

Comment:
why remove this? 

If we don't make this.textBlob to null, the textblob will be ref by this textblobmesh after it is recycled, therefore will never be released to gc in a long time

Reply:
TextBlob is changed to struct.